02-express-tutorial/routes/auth.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
```

Comment:
You don't necessarily need to keep all these files that get moved into the answers directory, but it's fine either way, up to you.